src/api/app/controllers/person/token_controller.rb

```diff
@@ -12,6 +12,10 @@ def index
 
     # POST /person/<login>/token
     def create
+      # TODO: remove when `trigger_workflow` feature is rolled out
+      raise(NoPermission, 'Not authorized to create a workflow token') if params[:operation] == 'workflow' &&
```

Comment:
Not sure this will help your customers to understand what is going on. You should tell them what's wrong no?

Reply:
I rephrased it. I mention the Beta program in the error message.